cpp/server/ct-server.cc

```diff
@@ -223,19 +235,14 @@ int main(int argc, char* argv[]) {
                                             &log_signer);
   LogLookup<LoggedCertificate> log_lookup(db);
 
-  // Make sure that we have an STH, even if the tree is empty.
   // TODO(pphaneuf): We should be remaining in an "unhealthy state"
   // (either not accepting any requests, or returning some internal
   // server error) until we have an STH to serve. We can sign for now,
   // but we might not be a signer.
-  SignMerkleTree(&tree_signer, &log_lookup);
+  thread signer(&SignMerkleTree, &tree_signer);
```

Comment:
Do we want/need a condition_variable/notification here to block until after the first run has completed?